packages/sdk/src/sdk/api/users/UsersApi.ts

```diff
@@ -251,7 +251,7 @@ export class UsersApi extends GeneratedUsersApi {
       action: Action.UPDATE,
       metadata: JSON.stringify({
         cid,
-        data: updatedMetadata
+        data: snakecaseKeys(updatedMetadata)
```

Comment:
small nit, would do this before line 244 so they are in the right casing when the cid is computed. While we are phasing it out, I can't be sure there isn't any weird EM logic that validates it.